docs/cloud/automation/terraform.md

```diff
@@ -1,2 +1,244 @@
-# Terraform provider
+---
+terraform_current_version: 1.5.1
+terraform_github_releases: https://github.com/EventStore/terraform-provider-eventstorecloud/releases/download
+---
 
+# Terraform provider 
+
+## Installation
+
+The current version of the provider is : {{ $frontmatter.terraform_current_version }}.
+
+The releases are available in Terraform's official [registry][terraform registry] and via [GitHub releases][terraform github releases].
+
+### Terraform 0.13+
+
+Terraform supports third party modules installed via the plugin registry.  
+Add the following to your terraform module configuration.
+
+
+<<< @/docs/cloud/automation/snippets/providers_eventstore.tf.hcl
+
+### Terraform 0.12
+
+In order for Terraform to find the plugin, the appropriate binary must be placed into the Terraform third-party plugin directory.  
+The location varies by operating system:
+
+- Linux and macOS `~/.terraform.d/plugins`
+- Windows `%APPDATA%\terraform.d\plugins`
+
+Alternatively, the binary can be placed alongside the main `terraform` binary.
+
+You can download the provider using the following commands:
+
+- Linux:
+
+<<< @/docs/cloud/automation/snippets/download_provider_linux.sh
```

Comment:
**QUESTION:** Shouldn't we also put information about other like `freebsd` or `arm64` versions?

Reply:
mmh, there is a link to the release page on github 
my best guess is those 3 are  the major platform used by customers ?

Reply:
Probably that should be fine. @mat-mcloughlin thoughts?